types/reactstrap/lib/Alert.d.ts

```diff
@@ -10,10 +10,9 @@ export interface UncontrolledProps {
   transitionLeaveTimeout?: number;
 }
 
-interface Props extends UncontrolledProps {
+export interface AlertProps extends UncontrolledAlertProps {
   isOpen?: boolean;
   toggle?: () => void;
 }
 
-declare var Alert: React.StatelessComponent<Props>;
-export default Alert;
```

Comment:
This is problematic because it means that deep imports to the package will now break, even if these were actually default exports.

I think these should be reverted, and the `default`s should continue to be re-exported as before.

Reply:
I didn't see anyone in importing reactstrap, but it's a good point. I'll go through and add all of the default exports back in. I'm going to wait until #21178 has been merged before I submit the commit.

Reply:
Should be merged!